lib/sqlalchemy/engine/row.py

```diff
@@ -123,7 +128,7 @@ def tuple(self) -> _TP:
         return self._tuple()
 
     @property
-    def _t(self) -> _TP:
+    def _t(self) -> Tuple[Unpack[_Ts]]:
```

Comment:
**Federico Caselli** (CaselIT) wrote:

I guess we can also deprecate this

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
Thanks, done